src/interfaces/pyo3/py_cover_crypt.rs

```diff
@@ -214,10 +167,14 @@ impl CoverCrypt {
         symmetric_key: &SymmetricKey,
         plaintext: Vec<u8>,
         authenticated_data: Option<Vec<u8>>,
-    ) -> PyResult<Vec<u8>> {
-        self.0
-            .encrypt(&symmetric_key.0, &plaintext, authenticated_data.as_deref())
-            .map_err(PyErr::from)
+        py: Python,
+    ) -> PyResult<PyObject> {
+        Ok(convert_to_pybytes(
+            &self
+                .0
+                .encrypt(&symmetric_key.0, &plaintext, authenticated_data.as_deref())?,
```

Comment:
This variable has been renamed, maybe go through your modifications to make sure to apply this renaming.
```suggestion
                .encrypt(&symmetric_key.0, &plaintext, authentication_data.as_deref())?,
```